pycontw/pycon.tw
# Update og:image of dashboard for 2021

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Update default og:image for 2021.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'MY PYCON page'
2. Check og:image update for 2021.

## Expected behavior
A clear and concise description of what you expected to happen.

## Related Issue
If applicable, refernce to the issue related to this pull request.

## More Information
**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)
